git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
Ok. What's the difference for those repos? Seems msys2 depends on mingw, am I right?